src/lib/topology/index.ts

```diff
@@ -98,18 +98,18 @@ export class Topology {
         this.em
       );
 
-      treeTop = bottom + NODE_MARGIN_BOTTOM;
+      treeLeft = right + NODE_MARGIN_BOTTOM;
```

Comment:
increasing the horizontal coordinate by the vertical component looks weird
is this correct?

Reply:
I decided to not introduce new constant. Maybe this should be renamed.

Reply:
If both horizontal and vertical margins are the same, it can as well be just `NODE_MARGIN`

Reply:
Fixed